### PR TITLE
chore(github): adding CI workflow github action

### PR DESCRIPTION
**Pull request by simonchaval**
## Describe your changes
Implementation of CI and unit tests for wiki module.
## Issue ticket number and link
[CO-1719](https://jira.support-ent.fr/browse/CO-1719)
## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

**Review by Sofianebernoussi — APPROVED** [review]

File: .github/workflows/dev-check-repository.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      image: opendigitaleducation/node:10-alpine
+      options: --user root -v ${{ github.workspace }}:/home/node/:rw
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run npm install
+        run: npm install
+      - name: Run build node with gulp
+        run: node_modules/gulp/bin/gulp.js build
+        #- name: Run test
```
Comment (simonchaval) [inline .github/workflows/dev-check-repository.yml:19]: Not implemented yet, this is why it's commented

File: .github/workflows/dev-check-repository.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    container:
+      image: gradle:4.5-alpine
+      options: --user root -v ${{ github.workspace }}:/home/gradle/:rw
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run build gradle
+        run: gradle assemble -g gradle-user-home
+        #- name: Run gradle test
```
Comment (simonchaval) [inline .github/workflows/dev-check-repository.yml:33]: Not implemented yet, this is why it's commented

File: .github/workflows/dev-check-repository.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      options: --user root -v ${{ github.workspace }}:/home/node/:rw
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run npm install
+        run: npm install
+      - name: Run build node with gulp
+        run: node_modules/gulp/bin/gulp.js build
+        #- name: Run test
+        #run: npm test
```
Comment (Matiyeu) [inline .github/workflows/dev-check-repository.yml:20]: Normal que ce soit commenté ?

Comment (simonchaval) [reply .github/workflows/dev-check-repository.yml:20]: Oui je l'ai mis en commentaire juste au-dessus.

**Review by Matiyeu — APPROVED** [review]

**Pull request merged**